<a href="https://colab.research.google.com/github/Riley-Hoang/3603-Programming-for-Data-Science/blob/main/Assignments/05-Foundations/02_markdown_and_formatting.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

## My Favorite Data Tools
- **Pandas** - for data manipulation and analysis
- **Matplotlib** - for data visualization
- **Scikit-learn** - for machine learning
- **Jupyter** - for interactive coding and documentation

## Top Programming Languages

1. **Python** - for data science, machine learning, and automation
2. **SQL** - for database querying and data manipulation
3. **JavaScript** - for web development and interactive visualizations


## Favorite Resources

**Favorite Website:** [Comics ](https://batcave.biz/) - Great for entertainment

**University Logo:**
![MSU Logo](https://brand.msu.edu/assets/images/msu-wordmark.png)

## Quadratic Formula

The quadratic formula is used to solve equations of the form $ax^2 + bx + c = 0$:

$$
x = \frac{-b \pm \sqrt{b^2 - 4ac}}{2a}
$$

Where:
- $a$, $b$, and $c$ are coefficients
- The discriminant $b^2 - 4ac$ determines the nature of the roots
- The $\pm$ symbol indicates there are two solutions

In [ ]:
# Create a list of 3 numbers and calculate their cubes
numbers = [2, 4, 6]
cubes = [n**3 for n in numbers]
print(f"Numbers: {numbers}")
print(f"Cubes: {cubes}")

Numbers: [2, 4, 6]
Cubes: [8, 64, 216]


## Analysis of Results

The original list contained the numbers **2, 4, and 6**. After computing the cube of each number (raising to the power of 3), we obtained:

- **2³ = 8**
- **4³ = 64**  
- **6³ = 216**

This demonstrates the rapid growth pattern of cubic functions, where each number's cube grows much faster than the original value. The relationship shows that as input values increase, their cubes increase at an even faster rate.